doc/api/fs.md

```diff
@@ -412,7 +414,15 @@ provided by the operating system's underlying directory mechanisms._
 added: REPLACEME
 -->
 
-* Returns: {AsyncIterator} to fully iterate over all entries in the directory.
+* Returns: {AsyncIterator} of {fs.Dirent}
+
+Asynchronously iterates over the directory via readdir(3) until all entries have
+been read.
+
+Entries returned by the async iterator are always an [`fs.Dirent`][].
+_(The `null` case from `dir.read()` is handled internally.)_
```

Comment:
Please remove either the `_` or the parentheses (or better yet, both--see below). The italics from `_` is supposed to be for emphasis. But putting something in parentheses is supposed to de-emphasize it. If it's important, emphasize it. If it's an aside, de-emphasize it.

Optional nit: I'd remove both the `_` and the parentheses. The sentence seems fine to stand by itself without being offset by both parentheses and typography.